features/storage/csi.feature

```diff
@@ -0,0 +1,38 @@
+Feature: CSI testing related feature
+
+  # @author chaoyang@redhat.com
+  # @case_id OCP-30787
+  @admin
+  Scenario: CSI images checking in stage env
+    Given the master version >= "4.4"
+    Given I switch to cluster admin pseudo user
+    Given admin uses the "csihostpath" project
```

Comment:
@chao007 thanks for explaining that the project `csihostpath` is setup during the cluster spin up with v3-launch-templates/functionality-testing/aos-4_3/hosts/storage/csihostpath/deploy.sh.  IMO, we should add a step to create the project if it's not defined so that the test does not rely on external shell setup script.   WDYT @akostadinov @liangxia ?

Reply:
These test cases are only to track the stage images due to errata owner does not check the stage images status. 
If the project does not exist, maybe there are some issues with the images. We need to check the related images manually.

Reply:
@pruan-rht   That script is involved in stage cluster installation. And the scenario is used to verify that.
If the project does not exist, it indicates there are issues which will need manual check involved, which is fine for me. Let me know if there are still concerns.